### PR TITLE
Role select option fixing

### PR DESCRIPTION
**Pull request by Tanosimboangy**
Hey team,
I've just completed fixing the styles about RoleSelectOption. So, feel free to check whenever you are free.
This is the [PR](https://github.com/onja-org/vamalaka-dong/compare/role_select_option_fixing?expand=1)

Refs: #41k83b
Clickup [Link](https://app.clickup.com/t/41k83b)

### CODE REVIEW

File: src/components/RoleIcon/Seller.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+interface SellerOption {
+  isMouseHover: any
+}
```
Comment (Mickaellah) [inline src/components/RoleIcon/Seller.tsx:5]: I think we cannot use any as a type.

Comment (Mickaellah) [inline src/components/RoleIcon/Seller.tsx]: I think we cannot use any as a type.

File: src/components/RoleSelectOption/RoleSelectOption.tsx [hunk 2]
```diff
@@ -1,159 +1,111 @@
...
+  line-height: 17px;
+  margin: 0;
+  margin-block-start: 4px;
+  @media (min-width: 375px) {
+    max-width: 239px;
+  }
 `
 
-export const Option: React.FC<OptionProps> = ({ label, text, src, alt }) => {
+export const Option: React.FC<OptionProps> = ({ label, text }) => {
```
Comment (andrzej-onja) [inline src/components/RoleSelectOption/RoleSelectOption.tsx]: Rename to RoleSelectOption

File: src/pages/RoleSelectOnboardOne/RoleSelectOnboardOne.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+import { PageFooter, Props } from '../../components/PageFooter/PageFooter'
+import { Header } from '../../components/Header'
+import { loggedIn } from '../../components/HeaderNavLink/HeaderNavLink'
+import { Option } from '../../components/RoleSelectOption/RoleSelectOption'
+import userIcon from '../../stories/assets/user.svg'
+import briefcaseIcon from '../../stories/assets/briefcase.svg'
+import { Login } from '../../components/Login/Login'
+import { LeftSide } from '../../components/LeftSide/LeftSide'
+
+export const RoleSelectOnboardOne: React.FC<Props> = ({ footerLinks }) => {
```
Comment (andrzej-onja) [inline src/pages/RoleSelectOnboardOne/RoleSelectOnboardOne.tsx]: Rename file name and component to OnboardingPage

File: src/pages/RoleSelectOnboardOne/RoleSelectOnboardOneStyles.tsx [hunk 4]
```diff
@@ -0,0 +1,136 @@
+import styled from 'styled-components'
```
Comment (andrzej-onja) [inline src/pages/RoleSelectOnboardOne/RoleSelectOnboardOneStyles.tsx]: Rename files to OnboardingPageStyles

File: src/components/RoleSelectOption/RoleSelectOption.tsx [hunk 5]
```diff
@@ -1,159 +1,111 @@
...
+    background: #f5f9ff;
+    border: 1px solid #041d42;
+    padding: 27px;
+    box-sizing: border-box;
+    box-shadow: 0px 4px 14px 1px rgba(0, 0, 0, 0.04);
+    border-radius: 6px;
+    ${RoleIcon} {
+      background-image: url(${darkPolygonBg});
+    }
+    ${Svg} {
```
Comment (andrzej-onja) [inline src/components/RoleSelectOption/RoleSelectOption.tsx]: How does this work?

Comment (andrzej-onja) [reply src/components/RoleSelectOption/RoleSelectOption.tsx]: Don't think `const Svg = styled.svg` is needed

File: src/components/RoleSelectOnboardOne/RoleSelectOnboardOne.stories.tsx [hunk 6]
```diff
@@ -1,73 +1,70 @@
-import { Story, Meta } from '@storybook/react'
-import {RoleSelectOnboardOne} from './RoleSelectOnboardOne'
-import { Props } from '../PageFooter/PageFooter'
+// import { Story, Meta } from '@storybook/react'
```
Comment (andrzej-onja) [inline src/components/RoleSelectOnboardOne/RoleSelectOnboardOne.stories.tsx]: This whole file should be removed

File: src/components/RoleSelectOnboardOne/RoleSelectOnboardOne.tsx [hunk 7]
```diff
@@ -1,69 +1,80 @@
...
-	MainContainer,
-	RoleOptionContainer,
-	Container,
-	AccountContainer,
-	HeaderContainer,
-	ImageContainer,
-	JoinUsHeader,
-	FooterContainer
-} from './RoleSelectOnboardOneStyles'
+// import React from 'react'
```
Comment (andrzej-onja) [inline src/components/RoleSelectOnboardOne/RoleSelectOnboardOne.tsx]: This whole file should be removed

File: src/components/RoleSelectOnboardOne/RoleSelectOnboardOneStyles.tsx [hunk 8]
```diff
@@ -1,127 +1,152 @@
-import styled from "styled-components"
+// import styled from 'styled-components'
```
Comment (andrzej-onja) [inline src/components/RoleSelectOnboardOne/RoleSelectOnboardOneStyles.tsx]: This whole file should be removed

File: src/components/RoleSelectOption/RoleSelectOption.tsx (hunk 5 above)
Comment (andrzej-onja) [reply src/components/RoleSelectOption/RoleSelectOption.tsx]: Any answers ???

File: src/components/RoleSelectOnboardOne/RoleSelectOnboardOne.stories.tsx (hunk 6 above)
Comment (andrzej-onja) [reply src/components/RoleSelectOnboardOne/RoleSelectOnboardOne.stories.tsx]: or renamed

File: src/components/RoleSelectOnboardOne/RoleSelectOnboardOne.tsx (hunk 7 above)
Comment (andrzej-onja) [reply src/components/RoleSelectOnboardOne/RoleSelectOnboardOne.tsx]: or renamed

File: src/components/RoleSelectOnboardOne/RoleSelectOnboardOneStyles.tsx (hunk 8 above)
Comment (andrzej-onja) [reply src/components/RoleSelectOnboardOne/RoleSelectOnboardOneStyles.tsx]: or renamed

File: src/components/RoleSelectOnboardOne/RoleSelectOnboardOne.tsx [hunk 9]
```diff
@@ -1,69 +1,64 @@
...
-						</div>
-					</RoleOptionContainer>
-				</MainContainer>
-			</TopContainerStyles>
-			<FooterContainer>
-				<PageFooter footerLinks={footerLinks} />
-			</FooterContainer>
-		</Container>
-	)
+export const RoleSelectOnboardOne: React.FC<Props> = () => {
```
Comment (andrzej-onja) [inline src/components/RoleSelectOnboardOne/RoleSelectOnboardOne.tsx]: Lets rename to just RoleSelectOnboard

File: src/components/RoleSelectOnboardOne/RoleSelectOnboardOneStyles.tsx [hunk 10]
```diff
@@ -1,80 +1,88 @@
...
+    max-width: 80%;
+    margin: auto;
+  }
+  @media (max-width: 850px) {
     & > div:nth-child(1) {
       display: none;
     }
-
+  }
+  @media (min-width: 850px) {
```
Comment (andrzej-onja) [inline src/components/RoleSelectOnboardOne/RoleSelectOnboardOneStyles.tsx]: please implement query helper as mentioned in https://onja20.slack.com/archives/C01BXQRM6LV/p1627044969006300

File: src/pages/GenericPages.tsx [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+      { linkContent: 'about', linkUrl: '#about' },
+      { linkContent: 'careers', linkUrl: '#careers' },
+      { linkContent: 'pricing', linkUrl: '#pricing' },
+      { linkContent: 'FAQ', linkUrl: '#faq' },
+    ],
+    id: '5',
+  },
+]
+
+const GenericPages: React.FC = ({ children }) => {
```
Comment (andrzej-onja) [inline src/pages/GenericPages.tsx]: Looks good just rename to GenericPage singular

File: src/components/RoleSelectOption/RoleSelectOption.tsx [hunk 12]
```diff
@@ -1,159 +1,107 @@
...
+const RoleSelectWrapper = styled.div`
+  z-index: 1;
+  padding: 28px;
+  cursor: pointer;
+  border-radius: 6px;
+  background: #ffffff;
+  box-shadow: 0px 2px 14px 1px rgba(0, 0, 0, 0.06);
+  @media (max-width: 320px) {
+    padding: 16px;
+  }
```
Comment (Mickaellah) [inline src/components/RoleSelectOption/RoleSelectOption.tsx]: You need to change this media query as well.

File: src/components/RoleSelectOption/RoleSelectOption.tsx [hunk 13]
```diff
@@ -1,159 +1,107 @@
...
+  justify-content: space-between;
   box-sizing: border-box;
-  border-radius: 6px;
+`
+const TextWrapper = styled.div`
+  margin: auto;
+  @media (max-width: 476px) {
+    max-width: 170px;
+    margin: auto;
+  }
```
Comment (Mickaellah) [inline src/components/RoleSelectOption/RoleSelectOption.tsx]: This one as well.

File: src/components/RoleSelectOption/RoleSelectOption.tsx [hunk 14]
```diff
@@ -1,159 +1,107 @@
...
+const RoleText = styled.p`
+  color: #979797;
+  font-size: 14px;
+  max-width: 161px;
+  line-height: 17px;
+  margin: 0;
+  margin-block-start: 4px;
+  @media (min-width: 375px) {
+    max-width: 239px;
+  }
```
Comment (Mickaellah) [inline src/components/RoleSelectOption/RoleSelectOption.tsx]: This is another one.

**Review by andrzej-onja — APPROVED** [review]

**Review by Mickaellah — APPROVED** [review]

**Review by Ny-Rianala — APPROVED** [review]

**Pull request merged**